### PR TITLE
[14.0][ADD] account_avatax_repair_oca

### PR DESCRIPTION
**Pull request by AlexPForgeFlow**

### CODE REVIEW

File: account_avatax_repair_oca/README.rst [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+- The module will check if there is a selected warehouse
+  and will automatically determine the address of the warehouse
+  and the origin location. If no address is assigned to the warehouse
+  the module will automatically use the address of the company as its origin.
+  Location code will automatically populate with the warehouse code
+  but can be modified if needed.
+
+- Hide Exemption & Tax Based on shipping address -- this will provide this
+  feature support at sale order level.
```
Comment (JordiBForgeFlow) [inline account_avatax_repair_oca/README.rst]: repair order level

File: account_avatax_repair_oca/README.rst [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+Usage
+=====
+
+The AvaTax module is integrated into Repair Orders and allows computation of taxes.
+Repairs order transactions do not appear in the in the AvaTax interface.
+
+The information placed in the repair order will automatically pass to the invoice
+on the Avalara server and can be viewed in the AvaTax control panel.
+
+They will be reported as a net deduction on the line item cost.
```
Comment (JordiBForgeFlow) [inline account_avatax_repair_oca/README.rst]: I don't understand this

File: account_avatax_repair_oca/README.rst [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+.. contents::
+   :local:
+
+Usage
+=====
+
+The AvaTax module is integrated into Repair Orders and allows computation of taxes.
+Repairs order transactions do not appear in the in the AvaTax interface.
+
+The information placed in the repair order will automatically pass to the invoice
```
Comment (JordiBForgeFlow) [inline account_avatax_repair_oca/README.rst:26]: I don't understand this

File: account_avatax_repair_oca/README.rst [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+=====================================================
+
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+
+|badge2|
+
+This module is a component of the Avatax Integration with odoo app.
```
Comment (JordiBForgeFlow) [inline account_avatax_repair_oca/README.rst:12]: Perhaps change this section

**Review by JordiBForgeFlow — APPROVED** [review]

**Review by mariadforgeflow — APPROVED** [review]
LGTM!

**Pull request merged**